### ISSUE TITLE
Every new sign-up creating new account

### ISSUE THREAD

**Issue by Oyshoboy** [scholtzm/meteor-accounts-steam#8]
Hi, hanging around meteor-accounts-steam, and everything is good, but everytime I sign up again - it's creates new account. How to handle this? Thank you!

**Comment by Oyshoboy** [comment]
Fro now resolved it by checking if user exist, and if it is - rewrite old one

**Comment by scholtzm** [comment]
I'm using these server hooks:

```
Accounts.onCreateUser((options, user) => { ... }); // fetch info from Steam API and attach it to user object
Accounts.onLogin((info) => { ... }); // update user's account in regular intervals
```

`onCreateUser` should only fire once per SteamID.

**Comment by Oyshoboy** [comment]
@scholtzm  so if the user created once, then it's should login automatically?
Strange, but I have every signUp - new meteor profile with the same steamId

**Comment by Oyshoboy** [comment]
@scholtzm ? In my each time you login with steam, onCreate method fires.

**Comment by scholtzm** [comment]
I cloned the example from here: https://github.com/scholtzm/meteor-accounts-steam-example

Slightly modified `server/main.js`:
![image](https://user-images.githubusercontent.com/2640934/36716266-f8919b90-1b99-11e8-9723-b1311caa639e.png)

```js
Accounts.onCreateUser(function(options, user) {
  console.log('>>> onCreateUser');
  console.log(options);
  console.log(user);

  return user;
});

Accounts.onLogin(function(info) {
  console.log('>>> onLogin');
  console.log(info);
});
```

I logged in via Steam, `onCreateUser` fired once, `onLogin` fired afterwards.
Then I cleared `localStorage` and logged in again. Only `onLogin` callback fired this time.

Everything works as expected.
